sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Microsoft.Azure.WebJobs.Extensions.ServiceBus.csproj

```diff
@@ -14,7 +14,6 @@
     <PackageReference Include="Microsoft.Azure.WebJobs.Sources" />
     <PackageReference Include="Microsoft.Azure.WebJobs" />
     <PackageReference Include="Microsoft.Extensions.Azure" />
-    <PackageReference Include="Azure.Messaging.ServiceBus" />
```

Comment:
Because this PR is leveraging some recent additions to the ServiceBus SDK that haven't been published to nuget yet, I'm temporarily referencing the project below. Before checkin, this PR will revert this change and will move to the official SB SDK package version.